qai/cleantext/cleantext.py

```diff
@@ -91,6 +89,34 @@ def run_pipeline(self, message):
                       self.elipses,
                       self.line_break)
 
+    def span_tokenize(self, message):
+        t = TreebankWordTokenizer()
+        tokens = []
+        for token, span in zip(t.tokenize(c), t.span_tokenize(c)):
```

Comment:
This code doesn't work. It fails with 

```sh
>>> import qai
>>> cleaner = qai.Cleantext()
>>> cleaner.span_tokenize('this is my message to you, rudy!!!')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/sh/q/repos/library.qai.utilities/qai/cleantext/cleantext.py", line 95, in span_tokenize
    for token, span in zip(t.tokenize(c), t.span_tokenize(c)):
NameError: name 'c' is not defined
```

I think you meant 
```suggestion
        for token, span in zip(t.tokenize(message), t.span_tokenize(message)):
```

Reply:
a [good editor](https://code.visualstudio.com/) would have told you that... 😜